File: mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int n;
    do
    {
        n = get_int("Height: ");
    } while (n <= 0);

    for (int row = 0; row < n; row++)
    {
        for(int left_side = 0; left_side < n; left_side++)
        {
            if (left_side >= (n - row - 1))
            {
                printf("#");
            }
            else
            {
                printf(" ");
            }
        }
        printf("  ");
        for (int right_side = 0; right_side <= row; right_side++){
            printf("#");
        }
        printf("\n");
    }
}

File: mario.py
import cs50

symbol = "#"
seperator = "  "

while True:
    height = cs50.get_int("Height: ")
    if height >= 1 and height <= 8:
        break


for i in range(1, height + 1):
    blank_spaces = " " * (height - i)
    symbols = symbol * i
    print(blank_spaces + symbols + seperator + symbols)
